guide/surveyor.md

```diff
@@ -0,0 +1,18 @@
+## Surveyor SRV-1
+
+%figure "Surveyor model in Webots"
+
+![model.png](images/robots/surveyor/model.png)
+
+%end
+
+Designed for research, education, and exploration, Surveyor's SRV-1 internet-controlled robot integrates a 1000MIPS 500MHz Analog Devices Blackfin BF537 processor, a digital video camera with resolution from 160x28 to 1280x1024 pixels, laser pointer ranging, and WLAN 802.11b/g networking on a quad-motor tracked mobile robotic base.
```

Comment:
It doesn't sounds like good English...

Missing article: **a** laser pointer. Not sure what means "ranging" here?

"WLAN 802.11b/g networking" &rarr; "a WLAN 802.11b/g network card"

"on" &rarr; "and"

Reply:
Not addressed.

Reply:
This was a copy-paste from a webpage, I entirely rewrote this paragraph.